--- contrib/init.d/rc.redhat.asterisk.user	2006-10-02 21:41:39.000000000 +0200
+++ contrib/init.d/rc.redhat.asterisk	2006-12-06 16:43:36.000000000 +0100
@@ -31,8 +31,8 @@
 # be sure that Asterisk's environment has permission to write the
 # files required  for  its  operation, including logs, its comm
 # socket, the asterisk database, etc.
-#AST_USER="asterisk"
-#AST_GROUP="asterisk"
+AST_USER="asterisk"
+AST_GROUP="asterisk"
 
 # Source function library.
 . /etc/rc.d/init.d/functions
