--- contrib/init.d/rc.redhat.asterisk	2007-06-01 17:06:03.000000000 -0500
+++ contrib/init.d/rc.redhat.asterisk.parche	2007-06-11 14:37:44.000000000 -0500
@@ -108,9 +111,50 @@
 	$DAEMON -rx 'reload' > /dev/null 2> /dev/null
 }
 
+chown_asterisk() {
+        echo SETTING FILE PERMISSIONS Asterisk
+        #chown -R asterisk:asterisk $ASTRUNDIR
+        #chown -R asterisk:asterisk /etc/asterisk
+        #chmod -R g+w /etc/asterisk
+        #chown -R asterisk:asterisk $ASTVARLIBDIR
+        #chmod -R g+w $ASTVARLIBDIR
+        #chown -R asterisk:asterisk $ASTLOGDIR
+        #chmod -R g+w $ASTLOGDIR
+        #chown -R asterisk:asterisk $ASTSPOOLDIR
+        #chmod -R g+w $ASTSPOOLDIR
+        #chown -R asterisk:asterisk $AMPWEBROOT/admin
+        #chmod -R g+w $AMPWEBROOT/admin
+        #chown -R asterisk:asterisk $FOPWEBROOT
+        #chmod -R g+w $FOPWEBROOT
+        #chown -R asterisk:asterisk $AMPWEBROOT/recordings
+        #chmod -R g+w $AMPWEBROOT/recordings
+        #chown -R asterisk:asterisk $AMPWEBROOT/_asterisk
+        #chmod u+x,g+x $ASTVARLIBDIR/bin/*
+
+        #if [ "$ASTAGIDIR" != "" ]; then
+        #        chmod u+x $ASTAGIDIR/*
+        #else
+        #        chmod u+x $ASTVARLIBDIR/agi-bin/*
+        #fi
+
+        #chmod u+x,g+x $AMPWEBROOT/admin/bounce_op.sh
+        #chmod u+x,g+x $FOPWEBROOT/*.pl
+        #chmod u+x $FOPWEBROOT/safe_opserver
+        chown asterisk /dev/tty9
+        # Ensure that various hardware devices are owned correctly.
+        [ -e /dev/zap ] && chown -R asterisk:asterisk /dev/zap
+        [ -e /dev/capi20 ] && chown -R asterisk:asterisk /dev/capi20
+        [ -e /dev/misdn ] && chown -R asterisk:asterisk /dev/misdn
+        [ -e /dev/dsp ] && chown -R asterisk:asterisk /dev/dsp
+
+        echo Permissions Asterisk OK
+}
+
+
 # See how we were called.
 case "$1" in
   start)
+	chown_asterisk
 	start
 	;;
   stop)
